hpat/io/_csv.cpp

```diff
@@ -309,14 +309,14 @@ static PyObject* csv_chunk_reader(std::istream * f, size_t fsz, bool is_parallel
         std::vector<size_t> line_offset = count_lines(f, hpat_dist_get_node_portion(fsz, nranks, rank));
         size_t no_lines = line_offset.size();
         // get total number of lines using allreduce
-        size_t tot_no_lines(0);
+        int64_t tot_no_lines = 0;
```

Comment:
why?

Reply:
Code readability and consistence. I think this is more common way to initialize variables. From the compiler point of view, in this particular place, they are equal.